### DISCUSSION TITLE
Rethink bounty system on Koda.

### DISCUSSION THREAD

**Discussion by VEER-TANMAY-SACHIN** [kodadot/nft-gallery#10500] [General]
### Is your feature request related to a problem?

I'm always frustrated when i see stale issues, i guess i have raised stalebot issue also.
I hope stalebot will be added.
For one issue i didn't got bounty, it creates a uncertainty with stale issues.

### Describe the solution you would like

I would like a stalebot, bounty issuer to discard and update bounties.

### Describe alternatives you have considered

I hope bounty program of KD isn't expired.

**Comment by vikiival** [comment]
Hey @TouchstoneTheDev,
thank you for opening a discussion.
I am in for having stalebot for issues.

But a rule of thumb is usually to pickup issues that were opened less than 3 months ago.

**Reply by VEER-TANMAY-SACHIN** [reply]
> Hey @TouchstoneTheDev, thank you for opening a discussion. I am in for having stalebot for issues.

stalebot is sure a thing to implement asap.
sometimes some ppl forget to close !

**Reply by VEER-TANMAY-SACHIN** [reply]
> Hey @TouchstoneTheDev, thank you for opening a discussion. I am in for having stalebot for issues.

https://github.com/kodadot/nft-gallery/pull/10507#issue-2363229488 get in for merging PR fr.

**Comment by VEER-TANMAY-SACHIN** [comment]
Fr ig that line should be in manifasto !
pardon me mate if i missed it
